test/Microsoft.ML.Tokenizers.Tests/Microsoft.ML.Tokenizers.Tests.csproj

```diff
@@ -35,6 +35,9 @@
     <None Update="Data\tokens_r50k_base.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="Data\cl100k_base.tiktoken">
```

Comment:
This a 1.6MB file being added to the repo. Are we ok with that?

Reply:
I deleted this file and made the test download it instead. We do the same in other tests.